config/rbac/mcp_server_get_role_binding.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: get-permissions
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: get-permissions
+subjects:
+  - kind: ServiceAccount
+    name: default
+    namespace: system
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)